source/deployment/ldap-group-sync.rst

```diff
@@ -0,0 +1,146 @@
+
+If you see a “Linked Failed” message, either click on the message or check the box before the group name to expose the inline link message, allowing you to try the link again.
+
+.. image:: ../images/LinkFailed.png
+
+Configure the group
+-------------------
+
+AD/LDAP groups that have been linked to Mattermost groups are able to be configured.  To configure the group, select “Configure”.  This will open up the Group Configuration page.  
+
+The Group Configuration page displays the group profile which includes the group name.  This name is automatically mapped from the AD/LDAP group common name attribute and is read-only.  
+
+Add default teams or channels for the group
+--------------------------------------------
+To add the default Team or Channels that you want the group members to default in, select either “Add Team” or “Add Channel” from the blue “Add Team or Channel” button.  You may select multiple teams and channels.  
+
+.. image:: ../images/Add_Team_Or_Channel.png
+
+Channels are nested below the Team they belong to in the team and channel list.  
+
+Teams that are open for anyone to join are indicated by:
+ 
+.. image:: ../images/open_team.png  
+   
+Teams that are not open for anyone to join are indicated by:
+ 
+.. image:: ../images/private_team.png 
+ 
+Public channels are indicated by: 
+ 
+.. image:: ../images/public_channel.png
+
+Private channels are indicated by:
+ 
+.. image:: ../images/private_channel.png  
+
+Teams added include default channels set in the `ExperimentalDefaultChannels config setting <https://docs.mattermost.com/administration/config-settings.html?highlight=configuration%20settings#default-channels-experimental>`_ (Town Square and Off-Topic). Adding a channel without setting the team adds the implied team to the listing below, but not to the group specifically.  Teams are listed in parentheses after the channel name in the channel selector.
+
+Teams and channels membership synchronization
+----------------------------------------------
+
+Default teams and channels will be added to the group member's user interface when they login for the first time after the group has been configured with the specified teams and channels. It may take a few seconds to load all team and channel memberships for a user depending on the number of teams and channels the group is defaulted to. In our testing, it took 6s for an organization with 200,000 users and 30,000 linked groups. This process is independent of the AD/LDAP synchronization process to support future functionality of groups.
+
+.. note::
+Users are not removed from the team or channel on subsequent synchronizations of the AD/LDAP groups.  Users will need to be manually removed from the team or channel per the existing functionality.  They will not be re-added if they were manually removed.
```

Comment:
A little confused by this section, writing out my understanding to see if it's correct. 

With the current functionality:  
- Removing a user from an LDAP Group does not remove them from the Mattermost Group, or the corresponding channels
- If you want to remove the user from the teams or channels, you can manually remove them
- If I manually remove a user from a channel, they will not be re-added to the channel even if they still belong to the Mattermost group 

Questions that come to mind as a reader: 
- Is removing them from channels planned future functionality? 
- Is the same thing true if users manually remove themselves from channels? 
- If the user is deactivated in LDAP, then do they lose access to the teams and channels? (I assume yes, it's just something that came to mind reading it)

Reply:
Added future improvement for automation of the removal and a reference to users removing themselves.  I added the note about deactivation in the next section (line 111)